src/base/base-mixin.js

```diff
@@ -668,6 +715,28 @@ export class BaseMixin {
         return result;
     }
 
+    _makeKeyboardAccessible (onClickFunction, ...onClickArgs) {
+        // called from each chart module's render and redraw methods
+        const tabElements = this._svg
+            .selectAll('.dc-tabbable')
+            .attr('tabindex', 0);
+                
+        if (onClickFunction) {
+            tabElements.on('keydown', (d, i) => {
+                // trigger only if d is an object undestood by KeyAccessor()
+                if (d3Event.keyCode === 13 && typeof d === 'object') {
```

Comment:
Maybe test if the key accessor returns truthy value? Just to be careful.

Reply:
I tried testing for it (`this.keyAccessor()(d)`), but it failed for bar and pie charts and `this.keyAccessor()` is always truthy because it returns a function. Is there another way?

Reply:
I agree, there probably is no better way. I am not sure that checking `typeof d === 'object'` provides all that much protection, since objects are pretty common in JS.

Were you seeing errors if you didn't check this?

Reply:
I thought you were on holiday! :) Yes. on charts like boxplot where you have a group with bound data and a clickable circle element. If you press enter on the circle, the event triggers, then propagates up to the group. If we don't check for `d` type, the `onClick` function will try to run twice and the first call (from circle) will error out with `TypeError: filters[0] is undefined`.

Reply:
Thanks. Okay, let’s stick with the object check for now. Maybe we will figure something out (like another CSS class?) later.